pkgs/development/tools/build-managers/sbt/default.nix

```diff
@@ -32,7 +32,9 @@ stdenv.mkDerivation (finalAttrs: {
     mkdir -p $out/share/sbt $out/bin
     cp -ra . $out/share/sbt
     ln -sT ../share/sbt/bin/sbt $out/bin/sbt
-    ln -sT ../share/sbt/bin/sbtn-x86_64-${
+    ln -sT ../share/sbt/bin/sbtn-${
```

Comment:
Or just use glob? (`sbtn-*`)

Reply:
You mean as the link target? There are all the binaries in the folder, for all architectures, that would not work.